packages/manager/apps/pci-object-storage/src/pages/objects/container/object/delete-replication/DeleteReplication.page.tsx

```diff
@@ -0,0 +1,180 @@
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { PciModal, useBytes } from '@ovh-ux/manager-pci-common';
+import { useNotifications } from '@ovh-ux/manager-react-components';
+import { OdsText } from '@ovhcloud/ods-components/react';
+import { useMemo } from 'react';
+import { Translation, useTranslation, Trans } from 'react-i18next';
+import {
+  createSearchParams,
+  useNavigate,
+  useParams,
+  useSearchParams,
+} from 'react-router-dom';
+
+import {
+  useAllStorages,
+  useStorage,
+  useStorageEndpoint,
+  useUpdateStorage,
+} from '@/api/hooks/useStorages';
+
+import { TContainer } from '../show/Show.page';
+import { useServerContainer } from '@/api/hooks/useContainer';
+import { useGetRegion } from '@/api/hooks/useRegion';
+
+export default function DeleteReplicationPage() {
+  const { t } = useTranslation(['containers/replication/delete']);
+
+  const { addSuccess, addError } = useNotifications();
+
+  const navigate = useNavigate();
+  const { projectId, storageId, replicationId } = useParams();
+  const decodedReplicationId = decodeURIComponent(atob(replicationId));
+
+  const [searchParams] = useSearchParams();
+
+  const { storage: targetContainer } = useStorage(
+    projectId,
+    storageId,
+    searchParams.get('region'),
+  );
+
+  const { url } = useStorageEndpoint(projectId, targetContainer);
+  const { formatBytes } = useBytes();
+
+  const { data: serverContainer, isLoading } = useServerContainer(
+    projectId,
+    searchParams.get('region'),
+    targetContainer?.name,
+    targetContainer?.id,
+  );
+
+  const { data: region } = useGetRegion(projectId, searchParams.get('region'));
```

Comment:
`searchParams.get('region')` could be undefined
Please check search params before invoking hooks or tou need to handle `undefined`  type in the hook itself